init.zsh

```diff
@@ -13,6 +13,7 @@ alias brewl='brew list'
 alias brewL='brew leaves'
 alias brewo='brew outdated'
 alias brews='brew search'
+alias brewS='brew service'
```

Comment:
The alias should be for `brew services` (instead of brew service, in singular), right? Not sure because I don't use this.

Reply:
Oh you're absolutely right,
sry about this